zerver/lib/actions.py

```diff
@@ -821,17 +821,18 @@ def do_send_typing_notification(notification):
     # type: (Dict[str, Any]) -> None
     recipient_user_profiles = get_recipient_user_profiles(notification['recipient'],
                                                           notification['sender'].id)
-    recipients = [{'id': profile.id, 'email': profile.email} for profile in recipient_user_profiles]
-    active_recipients = [profile for profile in recipient_user_profiles if profile.is_active]
-    sender = {'id': notification['sender'].id, 'email': notification['sender'].email}
+    active_recipient_ids = [profile.id for profile in recipient_user_profiles if profile.is_active]
```

Comment:
So we discussed this variable name--`active_recipient_ids`--a little bit on Zulip a couple of days ago, but I think it may have gotten lost in the shuffle.  I'd like to avoid "recipient_id" being part of the variable name here, because I think it will mislead folks into thinking that that id is for a Recipient model object, when it's actually a UserProfile object.  I think we can say `user_ids_to_notify` here.